packages/stitch/src/types.ts

```diff
@@ -46,6 +67,13 @@ export interface IStitchSchemasOptions<TContext = any> extends Omit<IExecutableS
   onTypeConflict?: OnTypeConflict;
   mergeDirectives?: boolean;
   mergeTypes?: boolean | Array<string> | MergeTypeFilter;
+  typeMergingOptions?: TypeMergingOptions;
+}
+
+export interface TypeMergingOptions {
+  typeDescriptionsMerger?: (candidates: Array<MergeTypeCandidate>) => string;
```

Comment:
This only does type descriptions, then? Field descriptions would go through the fieldConfigMerger?

Reply:
Yup. This could be improved further with an additional optional field description merger, but actually I'm not sure whether that would have to be instead of the field merger, or in addition, and if both are allowed, which one would operate first...